### PR TITLE
feat: [M3-6160] - Add resource links to Kubernetes empty state landing page

### PR DESCRIPTION
**Pull request by dwiley-akamai**
## Description 📝
- New utility file for empty state landing page constants and GA send event function
  - Refactors for DBaaS and Linode empty state landing pages related to the above
 - Resource links added to Kubernetes empty state landing page

## Preview 📷

Screenshot 2023-02-27 at 1 50 17 PM

## How to test 🧪
- Confirm that the GA events are still being sent as expected for the DBaaS and Linode empty state landing page interactions
  - This can be checked via network requests -- in Firefox, for example, you should see Google Analytics-related requests and you can drill down into the request and see the label, etc. associated with the click. In Chrome it seems the visibility of these requests depends on your local browser settings/configuration
- Confirm the GA events are being sent as expected for the Kubernetes empty state landing page interactions
- Confirm the links match the specs

### CODE REVIEW

File: packages/manager/src/features/Kubernetes/KubernetesLanding/KubernetesLandingEmptyState.tsx [hunk 1]
```diff
@@ -1,49 +1,183 @@
...
-        <a
-          href="https://www.linode.com/docs/applications/containers/kubernetes/how-to-deploy-a-cluster-with-lke/"
-          target="_blank"
-          rel="noopener noreferrer"
-          className="h-u"
-        >
-          Learn more about getting started with LKE.
-        </a>
-      </Typography>
+      <Typography variant="subtitle1">Placeholder</Typography>
```
Comment (dwiley-akamai) [inline packages/manager/src/features/Kubernetes/KubernetesLanding/KubernetesLandingEmptyState.tsx:178]: Adding the `Blocked` label on this PR until the placeholder copy is confirmed.

File: packages/manager/src/utilities/emptyStateLandingUtils.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { sendEvent } from 'src/utilities/ga';
+
+export const guidesMoreLinkText = 'Check out all our Docs';
+export const docsLink = 'https://www.linode.com/docs/';
+export const youtubeMoreLinkText = 'View our YouTube channel';
+export const youtubeChannelLink =
+  'https://www.youtube.com/playlist?list=PLTnRtjQN5ieb4XyvC9OUhp7nxzBENgCxJ';
+
+// retaining the old label for tracking
+export const youtubeMoreLinkLabel = 'View the complete playlist';
```
Comment (hkhalil-akamai) [inline packages/manager/src/utilities/emptyStateLandingUtils.ts:10]: @johnwcallahan do we still want to use the old label here? I know this was the case when I worked on this a bit ago but wanted to revisit.

**Review by hkhalil-akamai — APPROVED** [review]
LGTM. Tested and all links lead to the correct place/with the correct GA label.

**Review by jaalah-akamai — APPROVED** [review]
Looks good pending question 👍

File: packages/manager/src/utilities/emptyStateLandingUtils.ts (hunk 2 above)
Comment (jcallahan-akamai) [reply packages/manager/src/utilities/emptyStateLandingUtils.ts:10]: Yes that's still the case– good call out.

**Pull request merged**